platforms/ide/ide-plugins/build.gradle.kts

```diff
@@ -98,3 +98,5 @@ packageCycles {
      canHandleCi.cies/xinjd/.classpath
  */
 testFilesCleanup.reportOnly = true
+
+integTest.usesJavadocCodeSnippets = true
```

Comment:
Does this mean that the `autoTested` samples in the javadoc haven't been executed before, at least for the ide-plugins subproject?

Reply:
Not since they were moved to this subproject.

You can verify this by typing some nonsense in the sample code and running `:ide-plugins:build`.  Before this PR = SUCCESS, after = test failure.

Reply:
Got it. Thank you for fixing it!